website/content/docs/k8s/observability/metrics.mdx

```diff
@@ -0,0 +1,115 @@
+---
+layout: docs
+page_title: Metrics
+sidebar_title: Metrics
+description: Metrics for Consul on Kubernetes
+---
+
+# Metrics
+
+Consul on Kubernetes integrates with Prometheus and Grafana to provide metrics for Consul Service Mesh, including: service metrics, sidecar proxy metrics, Consul agent metrics, and gateway metrics. Service metrics can also be seen in the Consul UI Topology Visualization view. This section documents how to enable each of these.
```

Comment:
```suggestion
Consul on Kubernetes integrates with Prometheus and Grafana to provide metrics for Consul Service Mesh. The metrics available are:

* Service metrics
* Sidecar proxy metrics
* Consul agent metrics
* Ingress, Terminating and Mesh Gateway metrics

Service metrics can also be seen in the Consul UI Topology Visualization view. This section documents how to enable each of these.
```

* might be clearer with bullets
* should we link out to prometheus and grafana websites?
* what do we mean by service metrics?
* are service metrics viewable in the UI or are those sidecar metrics?

Reply:
Used these suggestions, thanks! And it is sidecar metrics not service, thanks for catching it!